bioptim/limits/constraints.py

```diff
@@ -329,7 +333,11 @@ def qddot_equals_forward_dynamics(_: Constraint, all_pn: PenaltyNodeList, **unus
             tau = nlp.states["tau"].mx if "tau" in nlp.states.keys() else nlp.controls["tau"].mx
 
             qddot = nlp.controls["qddot"].mx
-            qddot_fd = nlp.model.ForwardDynamics(q, qdot, tau).to_mx()
+            if with_contact:
+                model = biorbd.Model(nlp.model.path().absolutePath().to_string())
```

Comment:
Please add a todo or a comment to remind us to change this when we find an actual solution

Reply:
done.